config/cert-manager-prometheus/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
Probably re-use the existing dir `prometheus`, just name the service monitor file appropriately `cert-manager-monitor.yaml`.

Reply:
+1